assets/js/main.js

```diff
@@ -24,15 +32,58 @@ function initDatePicker() {
   }
 }
 
+function initSYAAnalyticsTrack() {
```

Comment:
There is no need to add Google Analytics (GA) here, we already have this set up via Google Tag Manager (GTM).